### PR TITLE
Removing MAS-SSO integration 

### PR DESCRIPTION
**Pull request by wtrocki**
This PR provides early integration giving us opportunity to refactor, discuss api and provide way to test early phases of integration.

# Changes

d10cca47041bbe39804d838eff7452e5b8bb944c Core changes that can be reviewed now 
a59d7557b53d6998fcf1feb65ccdaa1b5913034f Extra changes that are just deletion/irrelevant for review

### CODE REVIEW

File: pkg/cmd/serviceaccount/create/create.go [hunk 1]
```diff
@@ -154,7 +149,8 @@ func runCreate(opts *options) error {
 	creds := &credentials.Credentials{
 		ClientID:     serviceacct.GetClientId(),
 		ClientSecret: serviceacct.GetClientSecret(),
-		TokenURL:     cfg.MasAuthURL + "/protocol/openid-connect/token",
+		// TODO new location of the token url? From where we can take this value?
```
Comment (wtrocki) [inline pkg/cmd/serviceaccount/create/create.go:152]: CC @ziccardi @Rajagopalan-Ranganathan 

We need to discuss how CLI/UI will get new auth url:

What API will return it (maybe we can add that earlier than migration for end to end testing using still old value

Comment (wtrocki) [reply pkg/cmd/serviceaccount/create/create.go:152]: CC @akoserwal

File: pkg/core/auth/login/login.go [hunk 2]
```diff
@@ -37,22 +37,13 @@ type SSOConfig struct {
 }
 
 // Execute runs an Authorization Code flow login
-// enabling the user to log in to SSO and MAS-SSO in succession
+// enabling the user to log in to SSO
 // https://tools.ietf.org/html/rfc6749#section-4.1
-func (a *AuthorizationCodeGrant) Execute(ctx context.Context, ssoCfg *SSOConfig, masSSOCfg *SSOConfig) error {
+func (a *AuthorizationCodeGrant) Execute(ctx context.Context, ssoCfg *SSOConfig) error {
 	if err := a.loginSSO(ctx, ssoCfg); err != nil {
```
Comment (craicoverflow) [inline pkg/core/auth/login/login.go:43]: Simplify this to:

```go
return a.loginSSO(ctx, ssoCfg)
```

**Review by craicoverflow — COMMENTED** [review]
LGTM!

**Comment by wtrocki** [comment]
> But it might need to be dynamic?

Already addressed by the team. 
We going to get API that we will need to call on login to enable/disable mas-sso.
Then we will be able to merge that PR - it will probably take long time till we can

**Comment by MikeEdgar** [comment]
@wtrocki is there a build of this available? I'm interested in using this to test the admin API server Quarkus changes without MAS-SSO.

**Comment by wtrocki** [comment]
I give up on rebasing this PR. 2-3 weeks before SSO migration we will probably need to reapply the changes again to the latest codebase. CC @Rajagopalan-Ranganathan

**Comment by wtrocki** [comment]
FYI @rkpattnaik780

**Pull request closed without merging**